### PR TITLE
Assignment5a

### PR DESCRIPTION
**Pull request by AgarPiyush**

### CODE REVIEW

File: SpringProject/src/main/java/com/cnu2016/LoggerInterceptor.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    public boolean preHandle(HttpServletRequest request,
+                             HttpServletResponse response, Object handler) throws Exception {
+        long startTime = System.currentTimeMillis();
+        request.setAttribute("startTime", startTime);
+        return true;
+    }
+
+    @Override
+    public void postHandle(HttpServletRequest request, HttpServletResponse response, Object handler, ModelAndView modelAndView) throws Exception {
+        QueueConnect obj = new QueueConnect();
```
Comment (swapnilsm) [inline SpringProject/src/main/java/com/cnu2016/LoggerInterceptor.java:36]: Should connect only once.

File: SpringProject/src/main/java/com/cnu2016/Service/AmazonSQSClientclass.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by Piyush on 7/11/16.
+ */
+
+// TODO Autowire with QueueConnect
+
+@Component
+public class AmazonSQSClientclass {
```
Comment (swapnilsm) [inline SpringProject/src/main/java/com/cnu2016/Service/AmazonSQSClientclass.java:14]: Where is this used?

File: SpringProject/src/main/java/com/cnu2016/LoggerInterceptor.java (hunk 1 above)
Comment (AgarPiyush) [inline SpringProject/src/main/java/com/cnu2016/LoggerInterceptor.java:36]: Autowired it

File: SpringProject/src/main/java/com/cnu2016/Service/AmazonSQSClientclass.java (hunk 2 above)
Comment (AgarPiyush) [inline SpringProject/src/main/java/com/cnu2016/Service/AmazonSQSClientclass.java:14]: I was testing an alternative approach. Removed

**Pull request merged**